# channel statistics

std_msgs/Header header
int32  msg_received   # total number of msgs received over this channel
int32  msg_crc_error  # number of msgs with CRC failure
int32  msg_sent       # total number of msgs sent over this channel
